netci.groovy

```diff
@@ -285,6 +285,41 @@ def osShortName = ['Windows 10': 'win10',
 }
 
 // **************************
+// Define perf testing.  These tests should be run daily, and will run on Jenkins not in Helix
+// **************************
+[true, false].each { isPR ->
+    ['Release'].each { configurationGroup ->
+        ['Windows_NT'].each { os ->
+            def osGroup = osGroupMap[os]
+            def newJobName = "perf_${os.toLowerCase()}_${configurationGroup.toLowerCase()}"
+
+            def newJob = job(Utilities.getFullJobName(project, newJobName, isPR)) {
+                // On Windows we use the packer to put together everything. On *nix we use tar
+                steps {
+                    batchFile("call \"C:\\Program Files (x86)\\Microsoft Visual Studio 14.0\\VC\\vcvarsall.bat\" x86 && build.cmd /p:ConfigurationGroup=${configurationGroup} /p:OSGroup=${osGroup} /p:Performance=true")
+                    batchFile("C:\\Packer\\Packer.exe .\\bin\\build.pack .\\bin")
+                }
+            }
+
+            // Set the affinity.
+            Utilities.setMachineAffinity(newJob, os, 'latest-or-auto')
```

Comment:
Oh yeah for sure.  I wanted to first get the tests running and producing results.  Once we have that we can actually put them on the perf machines.  I just didn't want to do all of it at once.